benchmark/engine.go

```diff
@@ -79,7 +79,7 @@ func (b *BenchmarkEngine) Run() {
 						block := b.Block(uint8(myDifficulty))
 						end := time.Now()
 						block.DeleteFile()
-						diff := end.Sub(start)
+						diff := end.Sub(start).Seconds()
 						mu.Lock()
 						avg = append(avg, diff.Seconds())
```

Comment:
Redundant use of .Seconds(). Either use it on line 82 or 84

Reply:
Oh my bad, will fix